### ISSUE TITLE
Download URL returns error 404

### ISSUE THREAD

**Issue by kenokenobingo** [radicle-dev/homebrew-radicle#15]
Tarball is not available. When running `brew upgrade`, it returns:

```
==> Upgrading 3 outdated packages:
radicle-dev/radicle/radicle-alpha-ipfs v0.0.4 -> 0.0.4, radicle-dev/radicle/radicle-alpha v0.0.4 -> 0.0.4, imagemagick 7.0.8-47 -> 7.0.8-48
==> Upgrading radicle-dev/radicle/radicle-alpha-ipfs
==> Downloading http://storage.googleapis.com/static.radicle.xyz/releases/radicl

curl: (22) The requested URL returned error: 404 Not Found
Error: An exception occurred within a child process:
  DownloadError: Failed to download resource "radicle-alpha-ipfs"
Download failed: http://storage.googleapis.com/static.radicle.xyz/releases/radicle_0.0.4_x86_64-darwin.tar.gz
```

**Comment by MeBrei** [comment]
Hi @kenokenobingo, thank you for reporting this!

I merged a PR #16 that should fix this problem. Please let us know if you still experience an issue with the install.

**Comment by kenokenobingo** [comment]
Thanks, @MeBrei! It’s working now.
